lib/RTCSession.js

```diff
@@ -293,11 +294,12 @@ module.exports = class RTCSession extends EventEmitter
     {
       throw new Exceptions.InvalidStateError(this._status);
     }
+    const mediaConnection = this._connection;
 
-    // Check WebRTC support.
-    if (!window.RTCPeerConnection)
+    // Check support.
+    if (!isMediaConnection(mediaConnection))
```

Comment:
Let's do this check as soon as possible, on constructor.

Reply:
Moved